src/robusta/core/model/pods.py

```diff
@@ -24,6 +24,23 @@
     "Ei": 1024 * 1024 * 1024 * 1024 * 1024,
 }
 
+
+def format_unit(x: Union[float, int], /, *, base: Literal[1024, 1000] = 1024) -> str:
```

Comment:
Should this support value conversion for cpu and memory?
If so, I think there's an issue here.
For value `1.5` I get `1`

The other think is a matter of preference:
I think 
2441Ki should be 2.4Mi
2384Mi should be 2.3Gi

@Avi-Robusta @LeaveMyYard thoughts?